index.js

```diff
@@ -1,11 +1,12 @@
 import { WebhookClient, MessageEmbed, Formatters } from 'discord.js';
-import fetch from 'node-fetch';
+import { request } from 'undici';
 import cheerio from 'cheerio';
 import chalk from 'chalk';
 import { readFileSync, writeFileSync } from 'node:fs';
 import { setTimeout as setPromisedTimeout } from 'timers/promises';
 import { randomInt } from 'node:crypto';
 
+const configData = JSON.parse(readFileSync('./config.json', 'utf-8'));
```

Comment:
```suggestion
let configData = JSON.parse(readFileSync('./config.json', 'utf-8'));
```
Since it's being reassigned later on.

Reply:
whoopsies, thanks!